spec/models/organization_spec.rb

```diff
@@ -10,6 +10,12 @@
     end
   end
 
+  describe '#access_token' do
+    it 'selects a users access_token at random' do
```

Comment:
This test doesn't actually test what the description says.

`it 'returns an access_token'` would be more accurate.